mubench.pipeline/tasks/implementations/publish_findings.py

```diff
@@ -149,8 +160,15 @@ def _convert_graphs_to_files(potential_hit: Dict, findings_path: str) -> List[st
 
         return files
 
+    @staticmethod
+    def __get_potential_hit_size(potential_hit: 'SpecializedFinding') -> int:
+        return getsizeof(potential_hit) + sum([getsize(file) for file in potential_hit.files])
```

Comment:
Just for me to understand how this works: `getsizeof` calls `__sizeof__`, right? So the call above invokes the method declared below. If this is the case, we add the size of the dictionary (key + values), the size of the file list (i.e., the file names), and the size of the files themselves. To make this more obvious, could we move the `sum` to the `__sizeof__` method and replace the `__get_potential_hit_size` by `total_size(potential_hit)`?